svo (install me in module manager).xml

```diff
@@ -6425,11 +6412,12 @@ end</script>
 				<name>Check for updates</name>
 				<packageName></packageName>
 				<script>local downloadfolder = getMudletHomeDir().."/svo/downloads/"
+local baseUrl = "https://github.com/svof/svof/raw/in-client-svof/"
```

Comment:
Shouldn't this point out to the release zipped version that Travis CI builds instead of the raw files in the github repo?

Reply:
No. Quick and dirty and is using individual xmls. In the future I'd hope we can do individual file versioning, instead of updating the whole system with every update. This is a step towards that.